packages/trader/src/App/Components/Elements/ContractDrawer/__tests__/contract-drawer.spec.tsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import { createBrowserHistory } from 'history';
+import { render, screen } from '@testing-library/react';
+import { mockStore } from '@deriv/stores';
+import { Router } from 'react-router';
+import { isMobile, isDesktop } from '@deriv/shared';
+import ContractDrawer from '../contract-drawer';
+import TraderProviders from '../../../../../trader-providers';
+import userEvent from '@testing-library/user-event';
+
+const mocked_props = {
+    contract_info: '',
+    is_accumulator: false,
+    is_dark_theme: false,
+    is_market_closed: false,
+    is_multiplier: false,
+    is_vanilla: false,
+    is_smarttrader_contract: false,
+    is_sell_requested: false,
+    is_turbos: false,
+    onClickCancel: jest.fn(),
+    onClickSell: jest.fn(),
+    toggleHistoryTab: jest.fn(),
+} as unknown as React.ComponentProps<typeof ContractDrawer>;
+
+const contract_audit = 'Contract Audit';
+const contract_drawer_card = 'Contract Drawer Card';
+
+const withRouter = <T extends object>(Component: React.ComponentType<T>) => {
+    const history = createBrowserHistory();
+    const WrapperComponent = <U extends object>(props: T & U) => (
+        <Router history={history}>
+            <Component {...props} />
+        </Router>
+    );
+
+    return WrapperComponent;
+};
+const ContractDrawerComponent = withRouter(ContractDrawer);
+
+const default_mock_store = {
+    common: {
+        server_time: 123254362145 as unknown as moment.Moment,
```

Comment:
why are we asserting the type to moment?

Reply:
`server_time` has type `moment.Moment`, but I think I can use `toMoment()` from `@shared` package instead of asserting type

Reply:
yes please